### PR TITLE
Feature/rofe 4249 azure pipeline yml

### PR DESCRIPTION
**Pull request by twanarabo**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
+trigger:
+  branches:
+    include:
+      - master
+      - feature/*
+schedules:
+  - cron: 0 22 * * 0
+    branches:
+      include:
+        - develop
```
Comment (luismgoncalves) [inline azure-pipelines.yml:10]: As far as I can see, there is no develop branch on GitHub. So I'm sure how does this work

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+          SearchPattern: '**\bin\**\*.pdb'
+          PublishSymbols: false
+          SymbolServerType: TeamServices
+      - task: PublishBuildArtifacts@1
+        displayName: Publish Artifact
+        condition: succeededOrFailed()
+        inputs:
+          PathtoPublish: $(build.artifactstagingdirectory)
+          ArtifactName: $(BuildParameters.ArtifactName)
+          TargetPath: '\\my\share\$(Build.DefinitionName)\$(Build.BuildNumber)'
```
Comment (luismgoncalves) [inline azure-pipelines.yml:102]: what is this `\\my\share\` coming from?

Comment (twanarabo) [reply azure-pipelines.yml:102]: This actually comes form the visual pipeline that was created on Devops.

File: azure-pipelines.yml (hunk 1 above)
Comment (twanarabo) [reply azure-pipelines.yml:10]: I think this is a mistake. I will change it to master.

**Review by luismgoncalves — APPROVED** [review]

**Pull request merged**